__tests__/ngContainer/1.html

```diff
@@ -102,7 +102,8 @@
       <ng-container></ng-container>
     </div>
 
-    <div class="d-flex" *daPermission="'ACCOUNT_ADMIN'" [a.b]="t('13')">
+    <self-closing [a.b]="t('13')"/>
```

Comment:
Please add a self-closing tag for directive and pipe as well

Reply:
I was thinking of adding this in the next PR dedicated to Angular 17 compatibility, I had made a test suite dedicated to the self-closing tag.

Is it okay if we try to go step by step?

Reply:
Sure thing. remember to make dedicated commits in the next PR for better review 👍